database/migrations/2023_08_19_000000_update_staticdata_table.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
```

Comment:
Shouldn't the "Required Prod SQL Commands" be part of the migration step?

Reply:
I wasn't sure whether or not writing data in that way via a migration was considered a best practice. Would you advise I move that stuff into the migration file instead?

Reply:
That's probably a question for @luchaos. I'm worried that a manual step will be forgotten.

Reply:
I tend to go for commands that can be re-run instead, no data migrations in schema migrations.

Reply:
Had a lot of fun learning commands, what a useful feature.

In https://github.com/RetroAchievements/RAWeb/pull/1780/commits/64713552b4861a88617f1a4a22f9e92d6c7f6350, I've edited the PR description to remove all the SQL queries, and migrated them to a new command: `ra:platform:update-awards-static-data`.